left_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: base_left_wheel_joint

right_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: base_right_wheel_joint

caster_effort_controller:
  type: effort_controllers/JointEffortController
  joint: caster_joint

